PRODUCTION_SETUP.md

```
@@ -0,0 +1,236 @@
+kubectl get svc -A
+
+# Check application logs
+kubectl logs -n core-pipeline-prod -l app.kubernetes.io/name=core-pipeline
+kubectl logs -n dcmaidbot -l app.kubernetes.io/name=dcmaidbot
+```
+
+## 📊 Step 5: Access Your Applications
+
+After deployment, your applications will be available at:
+
+- **Authentik SSO**: https://auth.theedgestory.org
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)